scripts/us_census/pep/us_pep_sexrace/README.md

```diff
@@ -0,0 +1,74 @@
+# US Census PEP: Populations Estimates By Sex And Race
+
```

Comment:
Could you describe in the README regarding why we need so many different .py scripts for different year ranges?

Reply:
Added more description for different .py scripts for different year ranges in README.

Reply:
Sorry, I still feel a bit confused regarding why those .py scripts cannot be combined into a smaller set? Is it because different year's data has different schema? Or is it because different year's data needs different aggregations? Could you state the reason(s) clearly in README?

Reply:
Updated the README.md file.